languages/ruby/exercises/concept/hashes/.meta/design.md

```diff
@@ -0,0 +1,42 @@
+# Design
+
+## Goal
+
+The goal of this exercise is to teach the student the basics of the Concept of hashes through [Ruby][ruby-doc-hash].
+
+## Learning objectives
+
+After completing this exercise, the student should:
+
+- Know of the existence of the `Hash` collection.
+- Know how to create and initialise a `Hash`.
+- Know how to assign values to existing `Hash` keys.
+- Know how to add new `Hash` key.
+- Know how to iterate over a `Hash`.
+
+## Out of scope
+
+- Enumerable methos like select, reject, compact, etc.
+- Memory and performance characteristics.
+
+## Concepts
+
+- `Hash`: know of the existence of the `Hash` collection; know how to create and initialise a `Hash`; know how to assign values to existing `Hash` keys; know how to add new `Hash` key; know how to iterate over a `Hash`.
```

Comment:
Can or should we "bullet list" these concepts?

Reply:
This is the default way we list these.